streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -815,6 +825,168 @@ void to(final TopicNameExtractor<K, V> topicExtractor,
     <VR> KStream<K, VR> transformValues(final ValueTransformerWithKeySupplier<? super K, ? super V, ? extends VR> valueTransformerSupplier,
                                         final String... stateStoreNames);
 
+    /**
+     * Transform the value of each input record into zero or more new values (with possibly a new
+     * type) and emit for each new value a record with the same key of the input record and the value.
+     * A {@link ValueTransformer} (provided by the given {@link ValueTransformerSupplier}) is applied to each input
+     * record value and computes zero or more new values.
+     * Thus, an input record {@code <K,V>} can be transformed into output records {@code <K:V'>, <K:V''>, ...}.
+     * This is a stateful record-by-record operation (cf. {@link #mapValues(ValueMapper) mapValues()}).
```

Comment:
```suggestion
     * This is a stateful record-by-record operation (see {@link #mapValues(ValueMapper) mapValues()}).
```

Reply:
Throughout the javadocs in `KStreamImpl`, `cf.` is used. Do you want me to change all of them in a new PR? Are there other opinions on this?

Reply:
I don't know if we have a documentation style guide, but it's generally more accessible if we don't use Latin in our docs. No need to go making sweeping changes, but there's also no need to perpetuate an anti-pattern.

Reply:
> I don't know if we have a documentation style guide

I don't think we have one.

>  but it's generally more accessible if we don't use Latin in our docs.

This is surprising to me. I thought `cf.` is a standard abbreviation that everybody knows...

I would still use `cf.` in this PR for consistency reasons. If you prefer `see`, just create a ticket or open a new PR. We can also update this on a per-class basis, but I would prefer to keep it consistent within a class.